pkg/utils/common.go

```diff
@@ -168,6 +171,104 @@ var (
 	}
 )
 
+var (
+	NamespaceMetaGVK = metav1.GroupVersionKind{
```

Comment:
```suggestion
	NamespaceGVK = metav1.GroupVersionKind{
```

Reply:
suggest to rename all the following variables to have the meta in the middle to keep them consistent
eg, RoleMetaGVK